roles/socle-config/files/crd-conf-dso.yaml

```diff
@@ -249,6 +256,22 @@ spec:
                       description: The namespace for GitLab Operator.
                       type: string
                   type: object
+                gitlabCatalog:
+                  description: Configuration for GitLab Catalog.
+                  properties:
+                    catalogRepoUrl:
+                      description: GitLab Catalog repository url.
+                      default: https://github.com/cloud-pi-native/gitlab-ci-catalog.git
+                      type: string
+                    catalogBranchesApi:
```

Comment:
Est-ce que j'enlève cette partie jusqu'à la ligne 274 ?

Reply:
T'en as besoin pour pointer sur ton dépôt `catalog` local (ou accessible du moins) non ?

Reply:
Oui mais est-ce que c’est indispensable de récupérer toutes les branches et tags ou la branche master suffit ?

Reply:
La branche `main` suffit selon la politique qu'on applique, si on veut faire des releases un jour on fera une refacto pour chopper le dernier tag dispo.

Reply:
Ca marche, j'ai enlevé pour simplifier la PR pour le moment.

Reply:
En fait, je viens de me rendre compte d'un truc pour le role https://github.com/cloud-pi-native/socle/blob/main/roles/gitlab-catalog/tasks/main.yaml.
La task `- name: Create Catalog` jusqu'à la fin, ce serait équivalent à faire (si je résume un peu) :
```
git clone ${repoSource} --mirror
cd ${repoMirror}
git push ${repoTarget} --mirror
cd ..
rm -rf ${repoMirror}
```
Qu'en penses-tu @this-is-tobi ?